YGOProAnalyticsServer/Controllers/DecklistController.cs

```diff
@@ -74,20 +74,28 @@ public async Task<IActionResult> FindAll([FromQuery] DecklistBrowserQueryParamet
             }
 
             var decklists = await _decklistService.FindAll(
-                howManyTake: _config.DefaultNumberOfResultsPerBrowserPage,
-                howManySkip: _config.DefaultNumberOfResultsPerBrowserPage * (queryParams.PageNumber - 1),
                 minNumberOfGames: queryParams.MinNumberOfGames,
                 banlistId: queryParams.BanlistId,
                 archetypeName: queryParams.ArchetypeName,
                 statisticsFrom: statisticsFrom,
                 statisticsTo: statisticsTo);
 
+            var numberOfPages = Convert.ToInt32(
+                    Math.Ceiling(
+                        (double)(decklists.Count() / _config.DefaultNumberOfResultsPerBrowserPage)
```

Comment:
wrong implementation, make it  `((double)decklists.Count() / _config.DefaultNumberOfResultsPerBrowserPage)`